src/NHibernate/Cfg/MappingSchema/Hbm.generated.cs

```diff
@@ -427,10 +427,14 @@ public enum HbmCacheUsage {
         /// <remarks/>
         [System.Xml.Serialization.XmlEnumAttribute("transactional")]
         Transactional,
+        
+        /// <remarks/>
+        [System.Xml.Serialization.XmlEnumAttribute("never")]
+        Never,
```

Comment:
Here the "by hand" writtng of the code was also lacking the (almost) empty blank line. (I write "almost" because the generator does put spaces in it.)